gen/definitions/ext_community_set.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: Extcommunity Set
+path: Cisco-IOS-XR-um-route-policy-cfg:/routing-policy/sets/extended-community-opaque-sets/extended-community-opaque-set[set-name=%s]
+doc_category: Route Policy
+attributes:
+      - yang_name: set-name
```

Comment:
Let's make sure the indentation in YAML files is consistent (2 spaces):

```
attributes:
  - yang_name: set-name
```

Reply:
removed the extra space